clusterman/aws/aws_resource_group.py

```diff
@@ -204,3 +223,40 @@ def load(cls, cluster: str, pool: str, config: Any, **kwargs: Any) -> Mapping[st
     @classmethod
     def _get_resource_group_tags(cls) -> Mapping[str, Mapping[str, str]]:  # pragma: no cover
         return {}
+
+    def cached_describe_instances(self, instance_ids: Iterable[str]) -> Dict[str, InstanceDict]:
```

Comment:
This should probably just get called in the resource group initialization?  Since we reload resource groups on every autoscaling run, let's just store everything we need when it's initialized (see also https://github.com/Yelp/clusterman/blob/master/clusterman/aws/auto_scaling_resource_group.py#L59)

Reply:
(I'm actually a bit confused what's going on here... is this just describing all the instances in the resource group?  Why do we need to do that?)

Reply:
I think I was originally intending to determine EBS volume size from describe instances. But since we currently assume 300GB for all EBS-backed instances, I only need the instance type, which I get from `describe_spot_fleet_requests`. I can make this less of a hairball.